workflow/src/legenddataflow/scripts/merge_channels.py

```diff
@@ -24,138 +24,142 @@ def replace_path(d, old_path, new_path):
     return d
 
 
-argparser = argparse.ArgumentParser()
-argparser.add_argument("--input", help="input file", nargs="*", type=str, required=True)
-argparser.add_argument("--output", help="output file", type=str, required=True)
-argparser.add_argument(
-    "--in_db",
-    help="in db file (used for when lh5 files referred to in db)",
-    type=str,
-    required=False,
-)
-argparser.add_argument(
-    "--out_db",
-    help="lh5 file (used for when lh5 files referred to in db)",
-    type=str,
-    required=False,
-)
-argparser.add_argument(
-    "--channelmap",
-    help="channelmap",
-    type=str,
-    required=False,
-    default=None,
-)
-argparser.add_argument(
-    "--timestamp",
-    help="timestamp",
-    type=str,
-    required=False,
-)
-args = argparser.parse_args()
-
-# change to only have 1 output file for multiple inputs
-# don't care about processing step, check if extension matches
-
-channel_files = args.input.infiles if hasattr(args.input, "infiles") else args.input
-
-file_extension = Path(args.output).suffix
-
-if args.channelmap is not None:
-    channel_map = LegendMetadata(args.channelmap, lazy=True)
-    chmap = channel_map.channelmap(args.timestamp)
-else:
-    chmap = None
-
-if file_extension == ".dat" or file_extension == ".dir":
-    out_file = Path(args.output).with_suffix("")
-else:
-    out_file = args.output
-
-rng = np.random.default_rng()
-temp_output = f"{out_file}.{rng.integers(0, 99999):05d}"
-
-Path(args.output).parent.mkdir(parents=True, exist_ok=True)
-
-if file_extension in (".json", ".yaml", ".yml"):
-    out_dict = {}
-    for channel in channel_files:
-        if Path(channel).suffix == file_extension:
-            channel_dict = Props.read_from(channel)
-            fkey = ChannelProcKey.get_filekey_from_pattern(Path(channel).name)
-            if chmap is not None:
-                channel_name = f"ch{chmap[fkey.channel].daq.rawid:07}"
+def merge_channels() -> None:
+    argparser = argparse.ArgumentParser()
+    argparser.add_argument(
+        "--input", help="input file", nargs="*", type=str, required=True
+    )
+    argparser.add_argument("--output", help="output file", type=str, required=True)
+    argparser.add_argument(
+        "--in_db",
```

Comment:
```suggestion
        "--in-db",
```
One general comment: we should only use dashes in command line options. ArgParser automatically promotes them to underscores.